content/project/project_management.md

```diff
@@ -0,0 +1,196 @@
+* Facilitate asynchronous communication
+* Keep written records of staff communication
+* Organize communication records for easier recovery
+
+Communication platforms are usually chosen based on cultural norms -- again, it's important that team members feel comfortable communicating on the platform. OMSF project primarily use Slack, with some common norms:
+* Public channels for engaging with stakeholders
+* Private channels for high-volume staff communication
+* One Channel for each subproject / deliverable
+* One Channel for each group of stakeholders
+* One Channel for each software product
+* Use group DMs sparingly - named channels allow more flexibility
+
+OMSF projects also use Gmail for private, formal correspondence, including some engagement with industry partners.
+
+## Evaluation and metrics
+
+Simply put, metrics are measurable changes over time. More often than not, these are quantitative measurements that represent the state of a project. Metrics are almost never useful as individual points - they require a baseline and some time to determine health. Measuring Open Source Software is a hard problem, and OMSF projects have not settled on any specific set of metrics to track. We have considered
+* *Github statistics:* These primarily reflect the actions of the developer team, except in very widely used, mature projects.
+* *Package downloads:* Difficult to distinguish users from automated downloads in Continuous Integration pipelines.
+* *Citations:* Nonexistent for young projects. Difficult to normalize for mature projects, as software tools may be described in a number of publications, making it unclear which papers' citations should be included.
+
+## Publishing scientific papers
+
+If your software project is successful, it will lead to new, publication-worthy science. Publishing a scientific paper is a much different kind of task than software development, and requires different expertise and considerations. The following is adapted from [Open Force Field's writing process and stages](https://docs.google.com/document/d/1tvkGiP5N_T2v7iH_Y1R69V-pXJUnTS8RqEnwf_TG-wM/edit?usp=sharing)
+
+### Prepare a manuscript
+
+Assign a *lead author* and a *lead PI*. The *lead author* is responsible for:
+* providing drafts,
+* coordinating feedback (give deadlines!)
+* ensuring editing is done in a manner which does not result in irreconcilable editing clashes (e.g. multiple authors editing simultaneously in a way where they are making opposite edits simultaneously)
+The *lead PI* is responsible for 
+* supporting the lead author 
+* ensuring that papers progress through stages in a timely manner. 
+If an author does not provide feedback when required, the *lead PI* (or their delegate) should take responsibility to give that author appropriate options, such as (a) drop off of the paper, or (b) agree to let it proceed without providing review at that stage. 
```

Comment:
```suggestion

If an author does not provide feedback when required, the *lead PI* (or their delegate) should take responsibility to give that author appropriate options, such as (a) drop off of the paper, or (b) agree to let it proceed without providing review at that stage. 
```

Not sure on this one, but I don't think this paragraph is supposed to be part of the previous bullet point.